### PR TITLE
hotfixes - browser issues fixes

### PR DESCRIPTION
**Pull request by elisechant**

### CODE REVIEW

**Comment by elisechant** [comment]
screen shot 2016-10-12 at 4 56 41 pm

@catherine-jones seems like your tablet fix influences the KPIs on Overview

**Comment by catherine-jones** [comment]
Yes it should change kpi's on overview and show

**Comment by elisechant** [comment]
screen shot 2016-10-12 at 5 24 12 pm

labels need centering @catherine-jones

**Comment by elisechant** [comment]
@catherine-jones ah gotcha. duh. my bad.

**Comment by elisechant** [comment]
screen shot 2016-10-12 at 5 25 31 pm
screen shot 2016-10-12 at 5 25 28 pm
screen shot 2016-10-12 at 5 25 24 pm

wait. something weird happens at 767

**Comment by catherine-jones** [comment]
#TODO
- fix KPI's at width 767px [ ]
- fix KPI's titles and underline on ipad retina [ ]

File: client/src/dashboard/legacy/index.js [hunk 1]
```diff
@@ -280,7 +280,6 @@ $( document ).ready(function() {
     function toggleHighContrastMode(value) {
       console.log(value);
       if (value === "on") {
-        isHighContrastMode = true;
```
Comment (tobyhede) [inline client/src/dashboard/legacy/index.js]: Is this intentional?

File: client/src/dashboard/legacy/index.js [hunk 2]
```diff
@@ -280,7 +280,6 @@ $( document ).ready(function() {
     function toggleHighContrastMode(value) {
       console.log(value);
```
Comment (tobyhede) [inline client/src/dashboard/legacy/index.js]: The log is still here, but the isHighContrastMode switch has gone

File: client/src/dashboard/legacy/index.js (hunk 1 above)
Comment (elisechant) [inline client/src/dashboard/legacy/index.js]: ? no change

**Comment by elisechant** [comment]
@tobyhede fixed

**Pull request merged**